account_banking_pain_base/models/account_payment_order.py

```diff
@@ -509,10 +517,6 @@ def generate_remittance_info_block(self, parent_node, line, gen_args):
                     creditor_ref_info_type, 'Cd')
                 creditor_ref_info_type_code.text = 'SCOR'
                 # SCOR means "Structured Communication Reference"
-                creditor_ref_info_type_issuer = etree.SubElement(
-                    creditor_ref_info_type, 'Issr')
-                creditor_ref_info_type_issuer.text = \
-                    line.communication_type
```

Comment:
I'm not sure that this won't be problematic for other pain formats ? Why do you remove all this ?

Reply:
I removed it because the Issr tag was not present in the xsd file that defines pain.008.001.02.ch.03. So I thought it wasn't needed anymore, but I forgot that is was an OCA module. I will look to add it back (same for you following comments)